### DISCUSSION TITLE
Running Cypress in nested folder

### DISCUSSION THREAD

**Discussion by AbdallahHemdan** [cypress-io/cypress#18713] [General]
Currently, Our folder structure is as follows:

```md
- e2e
|  |_ config
|  |_ cypress
|  |_ .eslintrc.json
|  |_ .gitignore
|  |_ .prettierrc.json
|  |_ cypress.json
|  |_ tsconfig.json
- node_modules
- package.json
- ....
```

and my scripts for running cypress in my package.json is as follows:
```
"e2e:lint": "yarn eslint */**/*.ts  --fix",
"e2e:cy": "yarn cypress open",
"e2e:run": "yarn cypress run",
"e2e:cy-prod": "yarn cypress open --env configFile=production",
"e2e:run-prod": "yarn cypress run --env configFile=production",
"e2e:cy-dev": "yarn cypress open --env configFile=custom-development",
"e2e:run-dev": "yarn cypress run --env configFile=custom-development",
```

Each time I run the cypress command for `e2e:cy` it creates a new cypress examples folder in the root directory and does not run the tests inside my e2e folder, So my question is, How to run my scripts for the tests inside the e2e folder?

**Comment by AbdallahHemdan** [comment]
@jennifer-shehane Any recommendations? 🙏🏻

**Comment by bahmutov** [comment]
Use --project parameter to tell cypress to run in the e2e folder https://docs.cypress.io/guides/guides/command-line

"e2e:cy": "cypress open --project e2e"

**Reply by nokcy** [reply]
can you explain this a bit more please

**Comment by nokcy** [comment]
I am having a similar problem. Has anyone resolved this.
